### ISSUE TITLE
101-vm-user-image-data-disks template deployment is not valid

### ISSUE THREAD

**Issue by tuczap** [Azure/azure-quickstart-templates#1938]
After trying to deploy 101-vm-user-image-data-disks I get following error message:

> The template deployment 'Microsoft.Template' is not valid according to the validation procedure.

I am deploying in to existing resource group where my OS and Data disk images reside in single storage account. I input urls to os and data disks in this storage account.

Can anyone please verify if facing same issue?

**Comment by thekevinshen** [comment]
I'm getting the same error message as well. Have you found a way to solve it yet?

**Comment by tuczap** [comment]
Nope, but by using export template function in Azure and editing the template I was able to achieve the same thing this quickstart template does

**Comment by singhkay** [comment]
@tuczap  @thekevinshen Usually this is a generic error that is being improved to show up the actual cause. This is usually from the pre-deployment validation that checks for valid parameters. See if the dnsname and password parameter values are actually correct.
